src/PerCapita.jsx

```diff
@@ -35,7 +35,7 @@ export class PerCapita extends React.Component {
             onClick={ () =>
                 this.props.enablePer1000 &&
                 this._setNormalization( GEO_NORM_PER1000 ) }>
-            Complaints per 1,000 population
+            Complaints per 1,000
```

Comment:
Can we maintain the "population" language at higher resolutions? This would better match our existing landing page content and latest customer guidance. Let me know?

Reply:
Based on follow-up discussions, let's hold on this for now until we do some more testing / feedback. Thanks.